download/download.go

```diff
@@ -75,7 +75,7 @@ type downloaderResponse struct {
 }
 
 // New returns a new Downloader that can be started.
-func New(config Config, client rest.Client, path string) *Downloader {
```

Comment:
[nit] Technically, this is backwards-incompatible. We'll have to call that out in the release notes.